src/library/scala/collection/concurrent/TrieMap.scala

```diff
@@ -1037,8 +1037,8 @@ object TrieMap extends MapFactory[TrieMap] {
   }
 }
 
-
-private[collection] final class TrieMapIterator[K, V](var level: Int, private var ct: TrieMap[K, V], mustInit: Boolean = true) extends AbstractIterator[(K, V)] {
+// non-final as an extension point for parallel collections
+private[collection] class TrieMapIterator[K, V](var level: Int, private var ct: TrieMap[K, V], mustInit: Boolean = true) extends AbstractIterator[(K, V)] {
```

Comment:
deprecatedInheritance?

Reply:
I don't think that applies here since scala-parallel-collections isn't deprecated, and besides, it's `private[collection]`

Reply:
Let them cut and paste. (I haven't checked if it appears in internal API.) Just a thought. OK, back to work...